internal/pkg/constants/constants.go

```diff
@@ -9,12 +9,12 @@ var (
 
 const (
 	// PolicyServer CA Secret
-	PolicyServerTLSCert                  = "policy-server-cert"
-	PolicyServerTLSKey                   = "policy-server-key"
-	PolicyServerCARootSecretName         = "policy-server-root-ca"
-	PolicyServerCARootPemName            = "policy-server-root-ca-pem"
-	PolicyServerCARootCACert             = "policy-server-root-ca-cert"
-	PolicyServerCARootPrivateKeyCertName = "policy-server-root-ca-privatekey-cert"
+	PolicyServerTLSCert                = "policy-server-cert"
+	PolicyServerTLSKey                 = "policy-server-key"
+	KubewardenCARootSecretName         = "kubewarden-root-ca"
```

Comment:
I like it, better.
For audit-scanner args, this means we need to change the what secret we mount and the name of that secret in the helm-charts .

For helm upgrades, it may mean we need to recreate the policy-servers somehow, so they use the new secret. And we should also delete the now unused `policy-server-root-ca` in a post-install hook.